### DISCUSSION TITLE
button click disables the button unless I move the cursor

### DISCUSSION THREAD

**Discussion by seangenabe** [elkowar/eww#1219] [General]
When I click a button, I'm unable to trigger its :onclick action again, _unless_ I move the cursor a little bit.

I need to move the cursor a little bit in order to be able to trigger its onclick action again.

Example widget I have:

```yuck
      (button
        :onclick "swaync-client -t"
        "notifs"
      )
```

This isn't a bug report because: I'm not even sure if this is intended behavior.

**Comment by BishalK007** [comment]
Also facing the same issue.. nt sure why tho..
